src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBox/TextBox_BeforeTextChanging.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.TextBoxTests
+{
+	[SampleControlInfoAttribute("TextBox", "TextBox_BeforeTextChanging")]
```

Comment:
Why are you keeping the `Attribute` suffix here?

Reply:
Because I copy-pasted it from another sample, which kept the `Attribute` there. ;) I'll remove it.